Cannot spawn `fd_controller` and `joint_state_broadcaster`

To reproduce, run:

```shell
ros2 launch fd_bringup fd.launch.py
```

There's no topic `/fd/joint_states` and other topics, because `fd_controller` and `joint_state_broadcaster` are not spawned at all.

Related logs:

```
[controller_manager fd_controller-4] No executable found
[controller_manager joint_state_broadcaster-5] No executable found
[ERROR] [controller_manager joint_state_broadcaster-5]: process has died [pid 399582, exit code 1, cmd 'ros2 run controller_manager spawner--controller-manager /fd/controller_manager joint_state_broadcaster'].
[ERROR] [controller_manager fd_controller-4]: process has died [pid 399580, exit code 1, cmd 'ros2 run controller_manager spawner--controller-manager /fd/controller_manager fd_controller'].
```

<details>

<summary>Raw terminal log</summary>

```
❯ ros2 launch fd_bringup fd.launch.py
[INFO] [launch]: All log files can be found below /home/xxx/.ros/log/2023-12-10-00-14-42-815733-ortho-pc-399503
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [ros2_control_node-1]: process started with pid [399573]
[INFO] [robot_state_publisher-2]: process started with pid [399575]
[INFO] [static_transform_publisher-3]: process started with pid [399577]
[INFO] [controller_manager fd_controller-4]: process started with pid [399580]
[INFO] [controller_manager joint_state_broadcaster-5]: process started with pid [399582]
[static_transform_publisher-3] [WARN] [1702138482.998307331] []: Old-style arguments are deprecated; see --help for new-style arguments
[static_transform_publisher-3] [INFO] [1702138483.004720563] [static_transform_publisher]: Spinning until stopped - publishing transform
[static_transform_publisher-3] translation: ('0.000000', '0.000000', '0.000000')
[static_transform_publisher-3] rotation: ('-0.000000', '0.000000', '1.000000', '-0.000004')
[static_transform_publisher-3] from 'world' to 'fd_base'
[ros2_control_node-1] [WARN] [1702138483.009509122] [fd.controller_manager]: [Deprecated] Passing the robot description parameter directly to the control_manager node is deprecated. Use '~/robot_description' topic from 'robot_state_publisher' instead.
[ros2_control_node-1] [INFO] [1702138483.009611623] [resource_manager]: Loading hardware 'FDHapticInterface' 
[ros2_control_node-1] [INFO] [1702138483.010330306] [resource_manager]: Initialize hardware 'FDHapticInterface' 
[ros2_control_node-1] [INFO] [1702138483.010347027] [resource_manager]: Successful initialization of hardware 'FDHapticInterface'
[ros2_control_node-1] [INFO] [1702138483.010367332] [resource_manager]: 'configure' hardware 'FDHapticInterface' 
[ros2_control_node-1] [INFO] [1702138483.010370833] [resource_manager]: Successful 'configure' of hardware 'FDHapticInterface'
[ros2_control_node-1] [INFO] [1702138483.010373518] [resource_manager]: 'activate' hardware 'FDHapticInterface' 
[ros2_control_node-1] [INFO] [1702138483.010377351] [FDEffortHardwareInterface]: Starting ...please wait...
[robot_state_publisher-2] [INFO] [1702138483.022575046] [fd.robot_state_publisher]: got segment fd_base
[robot_state_publisher-2] [INFO] [1702138483.022648275] [fd.robot_state_publisher]: got segment fd_ee
[robot_state_publisher-2] [INFO] [1702138483.022652791] [fd.robot_state_publisher]: got segment fd_pitch_link
[robot_state_publisher-2] [INFO] [1702138483.022655216] [fd.robot_state_publisher]: got segment fd_roll_link
[robot_state_publisher-2] [INFO] [1702138483.022657246] [fd.robot_state_publisher]: got segment fd_x_link
[robot_state_publisher-2] [INFO] [1702138483.022659214] [fd.robot_state_publisher]: got segment fd_y_link
[robot_state_publisher-2] [INFO] [1702138483.022661062] [fd.robot_state_publisher]: got segment fd_yaw_link
[robot_state_publisher-2] [INFO] [1702138483.022663111] [fd.robot_state_publisher]: got segment fd_z_link
[controller_manager fd_controller-4] No executable found
[controller_manager joint_state_broadcaster-5] No executable found
[ERROR] [controller_manager joint_state_broadcaster-5]: process has died [pid 399582, exit code 1, cmd 'ros2 run controller_manager spawner--controller-manager /fd/controller_manager joint_state_broadcaster'].
[ERROR] [controller_manager fd_controller-4]: process has died [pid 399580, exit code 1, cmd 'ros2 run controller_manager spawner--controller-manager /fd/controller_manager fd_controller'].
[ros2_control_node-1] [INFO] [1702138483.910580225] [FDEffortHardwareInterface]: dhd : omega.3 device detected
[ros2_control_node-1] [INFO] [1702138483.910597458] [FDEffortHardwareInterface]: dhd : Rotation disabled 
[ros2_control_node-1] [INFO] [1702138483.910601390] [FDEffortHardwareInterface]: dhd : Effector Mass = 180.000000 (g)
[ros2_control_node-1] [INFO] [1702138483.911315563] [FDEffortHardwareInterface]: dhd : Gravity compensation enabled
[ros2_control_node-1] [INFO] [1702138483.911318385] [FDEffortHardwareInterface]: dhd : Device connected !
[ros2_control_node-1] [INFO] [1702138484.011613835] [FDEffortHardwareInterface]: System Successfully started!
[ros2_control_node-1] [INFO] [1702138484.011650471] [resource_manager]: Successful 'activate' of hardware 'FDHapticInterface'
[ros2_control_node-1] [INFO] [1702138484.013161741] [fd.controller_manager]: update rate is 1000 Hz
[ros2_control_node-1] [INFO] [1702138484.013239249] [fd.controller_manager]: RT kernel is recommended for better performance
```

</details>

--- a/fd_bringup/launch/fd.launch.py
+++ b/fd_bringup/launch/fd.launch.py
@@ -13,9 +13,13 @@
 # limitations under the License.
 
 from launch import LaunchDescription
-from launch.actions import DeclareLaunchArgument, ExecuteProcess
-from launch.substitutions import Command, FindExecutable
-from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import (
+    Command,
+    FindExecutable,
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
 
@@ -88,14 +92,16 @@ def generate_launch_description():
     load_controllers = []
     for controller in ['fd_controller', 'joint_state_broadcaster']:
         load_controllers += [
-            ExecuteProcess(
-                cmd=[
-                    'ros2 run controller_manager spawner' +
-                    f'--controller-manager /fd/controller_manager {controller}'
+            Node(
+                package='controller_manager',
+                executable='spawner',
+                namespace='fd',
+                arguments=[
+                    controller,
+                    '--controller-manager',
+                    '/fd/controller_manager',
                 ],
-                shell=True,
-                output='screen',
-            )
+            ),
         ]
 
     nodes = [

